lib/cdo/lesson_import_helper.rb

```diff
@@ -102,51 +96,45 @@ def self.create_activity_sections(activity_markdown, lesson_activity_id, levels)
 
     sections = []
     name = ''
-    position = 1
     sorted_matches.each do |match|
-      activity_section = nil
-      if match[:type] == 'skippable'
-        next
-      elsif match[:type] == 'tip'
+      if match[:type] == 'tip'
         activity_section = ActivitySection.new
         key = match[:match][3]&.strip || "#{match[:match][1]}-0"
         activity_section.tips = [create_tip(key, match[:match][1] || "tip", match[:match][4])]
         activity_section.key ||= SecureRandom.uuid
+        activity_sections = [activity_section]
         match[:activity_section_key] = activity_section.key
       elsif match[:type] == 'name'
         name = match[:match][1]
         next
-      elsif match[:type] == 'tiplink'
```

Comment:
Tip icons (tip links), however, might not be at the beginning of a line so we're going to handle them differently. This is the biggest change introduced in this PR, but it allows us to parse lines like:
1. `* tip!!!tip-0 my bulleted point`
1. `1. tip!!!tip-0 my numbered point`
1. `[slide] tip!!!tip-0 my sentence that has both a slide icon and a tip icon`

The third of these was introduced by the fact that we changed the slide icon syntax, the other big change in this PR. We could have shoehorned something in with the old logic but changing the logic gave us better parsing for the other two examples for free.